examples/blog-multiple-authors/sandbox.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "infiniteLoopProtection": true,
+  "hardReloadOnChange": false,
+  "view": "browser",
+  "template": "node",
+  "container": {
+    "port": 3000,
+    "startScript": "start",
```

Comment:
I think we want `dev` to be the recommended start script. We include `start` just as a friendly alias for people who are more used to that (but we might remove `start` at 1.0, maybe)